setup.py

```diff
@@ -1,31 +0,0 @@
-from setuptools import setup, find_packages
```

Comment:
When we import modules, we should first write system modules, and then other modules, in this case, import os should appear first.